api/src/middlewares/services/EmailService/SESService.ts

```diff
@@ -0,0 +1,93 @@
+import logger, { Logger } from "pino";
+import config from "config";
+import { SendRawEmailCommand, SESClient, SESServiceException } from "@aws-sdk/client-ses";
+import { createMimeMessage } from "mimetext";
+import { ses } from "../../../SESClient";
+import { FileService } from "../file";
+import { FormField } from "../../../types/FormField";
+import * as handlebars from "handlebars";
+import { ApplicationError } from "../../../ApplicationError";
+import { default as oathTemplate } from "./oathSubmissionTemplate";
+import { EmailService } from "./EmailService";
+export class SESService implements EmailService {
+  logger: Logger;
+  ses: SESClient;
+  fileService: FileService;
+  templates: any;
+
+  constructor({ fileService }) {
+    this.logger = logger().child({ service: "SES" });
+    this.ses = ses;
+    this.fileService = fileService;
+    this.templates = {
+      oath: SESService.createTemplate(oathTemplate),
```

Comment:
done this way so the app will immediately crash if the template compilation throws, instead of it crashing halfway down the line. 

i.e. when we push up to test, and then it crashes, we'll fix it first before sending over a build that would fail silently.